src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor

```diff
@@ -8,15 +8,28 @@
 
 <PageTitle><ApplicationName ResourceName="@nameof(Dashboard.Resources.ConsoleLogs.ConsoleLogsPageTitle)" Loc="@Loc" /></PageTitle>
 
-<div class="resource-logs-layout">
-    <h1 class="page-header">@Loc[nameof(Dashboard.Resources.ConsoleLogs.ConsoleLogsHeader)]</h1>
-    <FluentToolbar Orientation="Orientation.Horizontal">
+<AspirePageContentLayout AddNewlineOnDesktopToolbar="true">
```

Comment:
Mobile view has multiple scroll bars

![image](https://github.com/dotnet/aspire/assets/303201/4feea108-d8ee-4998-8fed-e0ce80a3d19a)

Reply:
That is not what appears for me
![image](https://github.com/dotnet/aspire/assets/20359921/91a2d470-38a6-4266-b63a-ca50439819ab)

Reply:
[It works on my machine](https://blog.codinghorror.com/the-works-on-my-machine-certification-program/) and marking this as resolved... isn't the best response here.

I'm using Chrome. I don't recognize what browser you're using. Does that explain the difference.

Reply:
I also get this in Edge:

![image](https://github.com/dotnet/aspire/assets/303201/ec075065-6a4a-4a1a-83c6-1e3181c830e6)

It appears to only happen in desktop view now.

Reply:
I am still unable to reproduce this on Edge. Please share the exact screen resolution you are using

Reply:
An unnecessary scrollbar is always visible for me. Browser size doesn't matter.

![console-logs-scrollbars](https://github.com/dotnet/aspire/assets/303201/32900236-2295-4290-ba9c-a5a6e08bd8db)

Reply:
Interestingly that doesn't happen on mac, but does on windows. Fixed - resolving. Please reopen if you find other issues.

<img width="1498" alt="image" src="https://github.com/dotnet/aspire/assets/20359921/c01e5f58-cbc6-420d-926d-3e99676c9ee1">